### ISSUE TITLE
UDFs may be in public schema on fresh install

### ISSUE THREAD

**Issue by marcocitus** [citusdata/citus#882]
Depending on whether you create extension or alter extension, some UDFs may be in a different schema.

```
postgres=# CREATE EXTENSION citus VERSION '5.2-1';
CREATE EXTENSION 
postgres=# ALTER EXTENSION citus UPDATE;
ALTER EXTENSION
postgres=# \df
                       List of functions
 Schema | Name | Result data type | Argument data types | Type 
--------+------+------------------+---------------------+------
(0 rows)
postgres=# CREATE EXTENSION citus;
CREATE EXTENSION
postgres=# \df
                                             List of functions
 Schema |             Name              | Result data type |          Argument data types          |  Type  
--------+-------------------------------+------------------+---------------------------------------+--------
 public | column_name_to_column         | text             | table_name regclass, column_name text | normal
 public | master_expire_table_cache     | void             | table_name regclass                   | normal
 public | master_get_new_placementid    | bigint           |                                       | normal
 public | worker_drop_distributed_table | void             | logicalrelid oid                      | normal
(4 rows)
```

**Comment by anarazel** [comment]
After fixing this we should add a regression test to the multi_extension.sql verifying that this isn't reintroduced. Should be fairly straightforward to do.

**Comment by anarazel** [comment]
@sumedhpathak Imo this should be fixed before 6.0 comes out.

**Comment by sumedhpathak** [comment]
Spoke with @ozgune and @anarazel Added 6.0 milestone to this task.
